recipes/slearn/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "slearn" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+
+source:
+  fn: slearn-0.2.4.tar.gz
+  url: https://files.pythonhosted.org/packages/f8/dd/3bd36708e78802bcab553e7b4bfc15a4a433e26fbe5f5911b1f2533fe8cd/slearn-0.2.4.tar.gz
```

Comment:
Can you use the canonical url shown in the example:

https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L16

Additionally the version specified above appears not to match the one you have in the url.

Reply:
Thanks Joshua, just replaced it.